samples/ControlCatalog.Android/ControlCatalog.Android.csproj

```diff
@@ -152,8 +156,16 @@
       <Name>ControlCatalog</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="HarfBuzzSharp">
+      <Version>2.6.1.5</Version>
+    </PackageReference>
+    <PackageReference Include="SkiaSharp">
```

Comment:
This should be added as .props file https://github.com/AvaloniaUI/Avalonia/blob/master/build/SkiaSharp.props